src/python/WMCore/Services/TagCollector/TagCollector.py

```diff
@@ -29,7 +30,8 @@ def __init__(self, url=None, logger=None, **kwargs):
         params.setdefault('cacheduration', 1)
         params['logger'] = logger if logger else logging.getLogger()
 
-        Service.__init__(self, params)
+        super(TagCollector, self).__init__(params)
+        self['logger'].debug("Initializing TagCollector with url: %s", self['endpoint'])
 
     def _getResult(self, callname="", clearCache=False,
```

Comment:
And it looks like we never pass any parameters to this method, so always relying on the default values.

Reply:
Good point, I just changed the code to look more like the CRIC service regarding this